skbio/metadata/_interval.py

```diff
@@ -559,6 +559,58 @@ def _reverse(self):
         # DONT' forget this!!!
         self._is_stale_tree = True
 
+    @classonlymethod
+    @experimental(as_of="0.5.0-dev")
+    def concat(cls, intvl_metadata):
+        '''Concatenate an iterable of ``IntervalMetadata`` objects.
+
+        Parameters
+        ----------
+        intvl_metadata : Iterable (IntervalMetadata)
+
```

Comment:
Possible to add one-liners to the parameters and returns? eg: "The interval metadata to concatenate"